content/maxwell3_fdem/emi_tutorial/induced_currents_body.rst

```diff
@@ -0,0 +1,94 @@
+.. _induced_currents_body:
+
+Induced currents in the body
+============================
+
```

Comment:
Here, a brief statement introducing the page, and outlining the scope (purpose), would help set the stage for the reader.